### PR TITLE
Draft page for graphic design and template related things.

### PR DESCRIPTION
**Pull request by jfy133**
A lot of this I wasn't really sure how to phrase (didn't want to make it too 'corporate' or 'branding'-y), so feel free to re-write.

Includes sections for:
- Logo
- Font
- Presentation templates (e.g. slideshows, posters etc)
- Workflow schematic examples

More examples will come from @ZandraFagernas once finalised.

One possible addition of examples to consider is the documentation figures from eager, as that also appears to be semi-popular, but Zandra and I would need to find more generic examples.

### CODE REVIEW

File: markdown/developers/graphic_design.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+## Colours
+
+The nf-core green colour is as follows:
+
+* RGB: 36,176,100
+* Hex: #24B064.
+
+## Presentation Templates
+
+nf-core regularly host [community events](https://nf-co.re/events), such as hackathons, and encourage training the wider bioinformatics community in creating nextflow and nf-core pipelines.s
```
Comment (ewels) [inline markdown/developers/graphic_design.md:29]:
```suggestion
nf-core regularly host [community events](https://nf-co.re/events), such as hackathons, and encourage training the wider bioinformatics community in creating nextflow and nf-core pipelines.
```

File: markdown/developers/graphic_design.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+The font(s) typically used in nf-core graphic design (particularly logos) is [Maven Pro](https://fonts.google.com/specimen/Maven+Pro), which is released under a Open Font License (see link for details).
+
+The exact variants of the fonts can also be seen at the [nf-core/logos GitHub repository](https://github.com/nf-core/logos).
+
+## Colours
+
+The nf-core green colour is as follows:
+
+* RGB: 36,176,100
+* Hex: #24B064.
```
Comment (ewels) [inline markdown/developers/graphic_design.md:25]:
```suggestion
* RGB: `36,176,100`
* Hex: `#24B064`
```

File: markdown/developers/graphic_design.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Presentation Templates
+
+nf-core regularly host [community events](https://nf-co.re/events), such as hackathons, and encourage training the wider bioinformatics community in creating nextflow and nf-core pipelines.s
+
+If you wish to introduce or run training sessions to your group or network, you can access a Google Doc template **here**.
```
Comment (ewels) [inline markdown/developers/graphic_design.md:31]:
```suggestion
<!--
If you wish to introduce or run training sessions to your group or network, you can access a Google Doc template **here**.
-->
```

Maybe can link to `usage/nf_core_tutorial`  here?

File: markdown/developers/graphic_design.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+Workflow diagram from [nf-core/sarek](https://github.com/nf-core/sarek/) and published in [Garcia _et al._ (2020, F1000 Research)](https://doi.org/10.12688/f1000research.16665.1) under a [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/) license.
+
+### Components
+
+Furthermore, there are some sub-components (such as icons) of the above example workflows made by members of the community, that are generally applicable to every pipeline. A couple of examples of these can be seen below, but are all accessible on the [nf-co.re website GitHub repository](https://github.com/nf-core/nf-co.re/assets/graphic_design/workflow_schematics/subcomponents), and can be re-used for your own graphics.
+
+<p align="center">
+    <img src="../../assets/graphic_design/workflow_schematics_components/sarek/single_fastq.png" alt="Single FASTQ Icon">
+    <img src="../../assets/graphic_design/workflow_schematics_components/sarek/multiple_vcf.png" alt="Multiple VCF Icon">
+</p>
```
Comment (ewels) [inline markdown/developers/graphic_design.md:56]: Maybe we can do this as a table instead, with links to the SVG for example..

Comment (jfy133) [reply markdown/developers/graphic_design.md:56]: Will get back to this later this week - but good idea! Do you mean as a markdown table?

**Comment by jfy133** [comment]
> Looking good! A few bits that could do with some work I think.
> 
> * Images should be `/public_html/assets`, not `/assets`.
>   
>   * You can move all of the files in `/assets` there and delete that folder.
> * Maybe we can link to the [Sarek page](https://nf-co.re/sarek) instead of embedding the overview?
>   
>   * Then no need to upload the graphic again, less likely to get out of sync.
>   * I think it's nice to keep pipeline-specific stuff in the pipeline repos basically..

Will move the files as suggested.

My idea was to actually display a few examples of the overviews so people can quickly compare to find the one they like best (rather than having to click through lots of links). But I will update the Sarek link to the website version.

**Comment by ewels** [comment]
You can still embed the overviews if you like, just hotlink the image from that repo. It's just having duplicates that I don't love as it will be tricky to keep them in sync if updated. Maybe make them quite small too, so that they don't take over the page..

File: markdown/developers/graphic_design.md (hunk 4 above)
Comment (ewels) [reply markdown/developers/graphic_design.md:56]: Yup 👍 There are a bunch of online markdown table generators that make this doable.

File: markdown/developers/graphic_design.md (hunk 3 above)
Comment (jfy133) [reply markdown/developers/graphic_design.md:31]: I could, but AFAICS this is just slides (either the HTML or PDF), not a template. Could you provide a template/original version someone can base their own tutorials off of? Or just basic image files from the tutorial that you think would be useful for others.

**Comment by jfy133** [comment]
> You can still embed the overviews if you like, just hotlink the image from that repo. It's just having duplicates that I don't love as it will be tricky to keep them in sync if updated. Maybe make them quite small too, so that they don't take over the page..

@ewels Should've made all the changes, other than an actual template for the nf-core tutorial slides (although I put a link to it in that section).

Let me know if this is what you meant. The workflow overviews were already directly linked to the pipeline's github repos; the specific components (the file icons in this case) are the ones I stored directly, they don't already exist on the Sarek page as independent files; I took them from Zandra's extraction when she used them for eager

**Comment by jfy133** [comment]
All fine with me!

**Comment by jfy133** [comment]
I think I have the Hackathon 'group update' Google slides from December? I could clean up one and share it back with you to store as a template if you wanted?

**Comment by ewels** [comment]
Yeah, the core team has all of this stuff from all of the hackathons. I’m not sure how useful it is though? Unless you’re actually organising a hackathon..

Regarding the tutorial, that and all of the other slide decks I make are in Apple KeyNote, which is why I usually just share them as PDFs.

I haven’t quite figured out what the main use case would be for all of this. I wonder if it would be best if I converted one of my short summary talks to PowerPoint and spiffed it up a bit specially for this purpose..? Then people can drop those 5 slides into whatever other talk that they are doing.

**Comment by jfy133** [comment]
> Yeah, the core team has all of this stuff from all of the hackathons. I’m not sure how useful it is though? Unless you’re actually organising a hackathon..
> 
> Regarding the tutorial, that and all of the other slide decks I make are in Apple KeyNote, which is why I usually just share them as PDFs.
> 
> I haven’t quite figured out what the main use case would be for the slide decks basically. I wonder if it would be best if I converted one of my short summary talks to PowerPoint and spiffed it up a bit specially for this purpose..? Then people can drop those 5 slides into whatever other talk that they are doing.

The last thing is sort of what I was thinking. Maybe run their own institutional hackathon (e.g. to convert all internal piplines to nf-core/nextflow), or present their new pipeline at a conference, and want to include some basic introductions or run similar hackathon-update presentations. I've had to introduce/explain nf-core a few times and my versions of your intro slides were often much inferior. Given you've all developed a pretty good community feeling already, I think having the 'brand' helps pull people in who are maybe a little unsure (e.g. during hackathons makes them feel they are a part of something 'bigger').

Up to you though!

**Pull request merged**